### PR TITLE
Feat : Added Video Listing Page

### PR DESCRIPTION
**Pull request by srejitk**
The following merge request adds the following features:

1. Header Component
2. Theme Provider
3. Video Listing Page
4. Video Database

### CODE REVIEW

File: src/pages/Explore/Explore.module.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+.videos_container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(14rem, 1fr));
+  /* flex-wrap: wrap;
+  justify-content: flex-start; */
```
Comment (nakulsharma15) [inline src/pages/Explore/Explore.module.css:5]: You can remove comments if it's no longer needed.

**Review by nakulsharma15 — APPROVED** [review]
Well Structured Code, Just a minor suggestion. I liked the way you named the variables and classes.

**Review by rahulcs754 — APPROVED** [review]
The code looks good and clean.

**Comment by srejitk** [comment]
Thank You

File: src/pages/Explore/Explore.module.css (hunk 1 above)
Comment (srejitk) [reply src/pages/Explore/Explore.module.css:5]: Will do that

**Pull request merged**